benchmark/atoi-corpus.h

```diff
@@ -119,17 +119,28 @@ struct CorpusStringLength {
     }
 };
 
+
+#if ZOO_CONFIGURED_TO_USE_AVX()
+#define AVX2_STRLEN_CORPUS_X_LIST \
+    X(ZOO_AVX, zoo::avx2_strlen)
+#else
+#define AVX2_STRLEN_CORPUS_X_LIST /* nothing */
```

Comment:
I am going to immediately add cases to this X list, you'll see this was the proper way to encapsulate this.

Do you have "bandwidth" to help me do the equivalent for ARM? basically I can give you a diff patch that enables ARM, and hopefully it'll work, but if it doesn't you could fix it, of course that I will be online to help if you want

Reply:
yup, i can add a patch if that helps. also feel free to push to this branch and i can pull and check it and fix errors i face